example/index.html

```diff
@@ -10,6 +10,34 @@
     <script defer src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/dart.min.js" integrity="sha512-14QR6tzX5xTNeMJKXzSK+xCquDvtNEr1jM5NlKy/149BBY50Kv70qqxHtzo6zClbtc1gIG7G0CGWXuMgPIMt0g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
     <script defer src="app.dart.js"></script>
     <title>Dart Markdown Live Editor</title>
+    <style>
+.gfm-color_chip span {
```

Comment:
Should open an issue on https://github.com/sindresorhus/github-markdown-css to request this be added.
Ideally and link the issue here!